content/7-webapp-features/1-fetch-data/angular/user.service.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { catchError, map, Subject } from 'rxjs';
+import { User } from './user';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class UserService {
+  error$: Subject<any> = new Subject();
```

Comment:
The `$` suffix is used to indicate what ?

Reply:
Is to indicate that the variable is a data stream.